builtin/credential/cert/path_certs.go

```diff
@@ -124,18 +137,26 @@ func (b *backend) pathCertRead(
 		return nil, nil
 	}
 
-	duration := cert.TTL
-	if duration == 0 {
-		duration = b.System().DefaultLeaseTTL()
+	ttl := cert.TTL
+	if ttl == 0 {
+		ttl = b.System().DefaultLeaseTTL()
```

Comment:
When returning the cert info we should return the *actual* stored info, not the *effective* info. Let the cert.TTL and cert.MaxTTL bits stay at zero if they're zero, because that indicates that they weren't configured.

Reply:
Ok, I'd guess the same applies for max_ttl on https://github.com/hashicorp/vault/pull/3642/files/4043a5f8dda025e58ab7c399126f3a7a9b229396#diff-881681caf9da9a575287a64096baa902R146

Reply:
Yes.